setup.py

```diff
@@ -9,16 +9,37 @@
 
 directory = os.path.abspath(os.path.dirname(__file__))
 
-
-with open(os.path.join(directory, "requirements.txt")) as f:
-    contents = f.read()
-    REQUIREMENTS = [i.strip() for i in contents.strip().split("\n")]
+extras = ["orjson", "playground", "uvloop"]
 
 
 with open(os.path.join(directory, "README.md")) as f:
     LONG_DESCRIPTION = f.read()
 
 
+def parse_requirement(req_path):
+    """Parse requirement file.
+    Example:
+        parse_requirement('requirements.txt')       # requirements/requirements.txt
+        parse_requirement('extras/playground.txt')  # requirements/extras/playground.txt
+    Returns:
+        List[str]: list of requirements specified in the file.
+    """
```

Comment:
Should we move the type annotations to the function body?

```python
def parse_requirement(req_path: str) -> List[str]:
```

In Piccolo the docstrings tend to start with a newline character - it might be worth changing this to make it consistent? Not a big deal though.

```python
"""
Parse requirement file.
```